static/edx-pl/sass/main-ltr.scss

```diff
@@ -0,0 +1,22 @@
+// ------------------------------
```

Comment:
@feanil, good question, for now these main-ltr/rtl.scss files aren't used or referenced directly. Instead the Web Certs UI has its own app-centric compile (see https://github.com/edx/edx-platform/blob/talbs/certs-web-ui/lms%2Fstatic%2Fcertificates%2Fsass%2Fmain-ltr.scss and https://github.com/edx/edx-platform/blob/talbs/certs-web-ui/lms%2Fstatic%2Fcertificates%2Fsass%2Fmain-rtl.scss) and imports a lot of these same partials noted here. The output CSS (which is compiled to https://github.com/edx/edx-platform/blob/talbs/certs-web-ui/lms%2Fstatic%2Fcertificates%2Fcss%2Fmain-ltr.css and https://github.com/edx/edx-platform/blob/talbs/certs-web-ui/lms%2Fstatic%2Fcertificates%2Fcss%2Fmain-rtl.css) seem like the files we need to track from a pipeline/collectstatic PoV, right?

(Apologies for any confusion, I just pointed you two to this file since it had a lot of references to the partials I've added and the app is using.)